tests/integration/add.js

```diff
@@ -164,6 +164,33 @@ test('adds multiple objects to db', function (t) {
   .catch(t.error)
 })
 
+test('fail validation adding multiple objects to db', function (t) {
+  t.plan(1)
+
+  var name = uniqueName()
+  var store = new Store(name, {
+    PouchDB: PouchDB,
+    remote: 'remote-' + name,
+    validate: function () { throw new Error() }
```

Comment:
can you pass in an error message here

Reply:
Will do. But, how does it impact our tests?

Reply:
you can check if the error message you throw arrives in the `.catch(error) {}`, see my comment below: https://github.com/hoodiehq/hoodie-store-client/pull/165#discussion_r135302205